pkg/wekafs/wekafs.go

```diff
@@ -373,10 +402,108 @@ func (driver *WekaFsDriver) Run(ctx context.Context) {
 
 	}()
 
+	// Wait for manager cache to sync before starting GRPC server
```

Comment:
here exactly you can take it out as a method and call it in `Probe` but only for controller and if total capacity enforced